README.md

```diff
@@ -14,6 +14,22 @@ ember install ember-feature-flags
 
 ### Usage
 
+#### Computed-Property Macro
+
+This addon provides a computed-property macro that you can use to add properties that toggle
+whenever feature-flags do:
+```js
+import Controller from '@ember/controller';
+import { macro as featureFlag } from 'ember-feature-flags';
```

Comment:
Nitpick on the name. Can we come up with something that won't want to be aliased as often as `macro`?